### ISSUE TITLE
get blurry image after lora finetune

### ISSUE THREAD

**Issue by hanwenxu1** [THUDM/CogVideo#458]
I'm trying to use `train_cogvideox_lora.py` to finetune 2b model with customized dataset. When I set the train lr is 1e-5, and the epoch is 2, I get some normal result.
![image](https://github.com/user-attachments/assets/65102cbd-7e42-4e7d-8a43-091fc4a38d7c)
Then I found some advice that the lr of lora should be larger, like 5e-4 to 1e-3. I tried to set lr as 5e-4 and the epoch is 10. But I get worse results.
![image](https://github.com/user-attachments/assets/63fe9a03-6d96-4b82-9925-e8ed6a1c52a4)
![image](https://github.com/user-attachments/assets/36b4ee3a-8797-49fe-aae7-310909b0fd7d)
I wonder that why it became worse and it seemed so vague.
The train loss is here:
![image](https://github.com/user-attachments/assets/b8041591-e3e1-49fb-89da-b269509c245c)

**Comment by zRzRzRzRzRzRzR** [comment]
We systematically updated the fine-tuning code. This version fixes some minor algorithmic discrepancies. It may effectively solve the problem.
